support/mock/mockrt.go

```diff
@@ -392,18 +431,53 @@ func (rt *Runtime) Reset() {
 
 // Calls f() expecting it to invoke Runtime.Abort() with a specified exit code.
 func (rt *Runtime) ExpectAbort(expected exitcode.ExitCode, f func()) {
+	prevState := rt.state
+
 	defer func() {
 		r := recover()
+		if r == nil {
+			rt.t.Errorf("expected abort with code %v but call succeeded", expected)
+			return
+		}
 		a, ok := r.(abort)
 		if !ok {
 			panic(r)
 		}
 		if a.code != expected {
 			rt.t.Errorf("abort expected code %v, got %v %s", expected, a.code, a.msg)
```

Comment:
I don't see any reason to continue execution (`Errorf`) if either of these assertions fail. Can we use `Fatalf` here?